src/Raven.Server/Json/BlittableNumber.cs

```diff
@@ -24,6 +24,14 @@ public static NumberParseResult Parse(object value, out double doubleResult, out
                 return NumberParseResult.Double;
             }
 
+            if (value is float)
+            {
+                longResult = long.MinValue;
+                doubleResult = (double)(decimal)(float)value;
```

Comment:
Actually this is not a good solution. This will overflow when float.MaxValue.
Is there a reason why we aren't using float?